### PR TITLE
EOS-22267: requirement update - support user to login directly inside nodecli

### PR DESCRIPTION
**Pull request by divya-vijay**
Signed-off-by: Divya Vijayakumar <divya.vijaykumar@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: lr-cli/cortx_setup/commands/node/finalize.py [hunk 1]
```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
                     targets=node_id
                 )
 
+            else:
+                self.logger.debug(f"Ensuring {user} user permissions on system")
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.bashrc',
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/node/finalize.py:224]: Can we add alias to profile instead? Lets not make it bash specific

File: lr-cli/cortx_setup/commands/node/finalize.py [hunk 2]
```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
...
+                    ),
+                    targets=node_id
+                )
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.profile',
+                        text=('sudo /usr/bin/nodecli \n'
+                              f'source {home_dir}/.bashrc')
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/node/finalize.py:235]: Do we really need to source bashrc?

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/node/finalize.py (hunk 1 above)
Comment (divya-vijay) [reply lr-cli/cortx_setup/commands/node/finalize.py:224]: Alias for specific commands in profile doesn't work.

File: lr-cli/cortx_setup/commands/node/finalize.py (hunk 2 above)
Comment (divya-vijay) [reply lr-cli/cortx_setup/commands/node/finalize.py:235]: yes, sourcing rc file on profile login ensures our alias changes are in place.

File: lr-cli/cortx_setup/commands/node/finalize.py [hunk 3]
```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
...
                 )
 
+            else:
+                self.logger.debug(f"Ensuring {user} user permissions on system")
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.bashrc',
+                        text='alias nodecli="sudo nodecli"'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/node/finalize.py:225]: Change this to:
`alias nodecli="sudo /usr/bin/nodecli"`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/node/finalize.py (hunk 3 above)
Comment (divya-vijay) [reply lr-cli/cortx_setup/commands/node/finalize.py]: done!

**Review by nitin-seagate — APPROVED** [review]
Looks good

**Review by sradhanand-pati — APPROVED** [review]

**Pull request merged**